content/docs/components-and-props.md

```diff
@@ -76,30 +76,30 @@ ReactDOM.render(
 );
 ```
 
-[](codepen://components-and-props/rendering-a-component)
+[**Essayer sur CodePen**](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+Récapitulons ce qui se passe dans cet exemple:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. On appelle `ReactDOM.render()` avec l'élément `<Welcome name="Sara" />`.
+2. React appelle le composant `Welcome` avec comme props `{name: 'Sara'}`.
+3. Notre composant `Welcome` retourne un élément `<h1>Bonjour, Sara</h1>` pour résultat.
+4. Le DOM React met à jour efficacement le DOM de manière à correspondre à `<h1>Bonjour, Sara</h1>`.
 
->**Note:** Always start component names with a capital letter.
+>**Note:** Commencez toujours vos noms de composants par une majuscule.
 >
->React treats components starting with lowercase letters as DOM tags. For example, `<div />` represents an HTML div tag, but `<Welcome />` represents a component and requires `Welcome` to be in scope.
+>React concidère les composants commençant par des lettres minuscules comme des balises DOM. Par exemple, `<div />` représente une balise HTML div, mais `<Welcome />` représente un composant et nécessite `Welcome` pour appartenir aux contexte d'exécution.
```

Comment:
"représente un composant et nécessite `Welcome` **pour** appartenir aux contexte d'exécution."
Not sure but i think " nécessite que 'Welcome' appartienne au(no x) au contexte d'execution" seems more appropriate